docs/content/docs/build/auth.md

```diff
@@ -0,0 +1,112 @@
+JWT. The Azure Fluid Relay service uses signed JWTs for establishing trust with calling clients.
+
+The next question is: what data should you send?
+
+You need to send your *tenant ID* so that FRS can look up the right secret key to validate your request. You need to
+send the *container ID* (called `documentId` in the JWT) so FRS knows which container the request is about. Finally, you
+need to also set the *scopes (permissions)* that the request is permitted to use -- this allows you to establish your
+own user permissions model if you wish.
+
+```json {linenos=inline,hl_lines=["5-6",9]}
+{
+  "alg": "HS256",
+  "typ": "JWT"
+}.{
+  "documentId": "azureFluidDocumentId",
+  "scopes": [ "doc:read", "doc:write", "summary:write" ],
+  "user": {
+    "name": "TestUser",
+    "id": "Test-Id-123"
+  },
+  "iat": 1599098963,
+  "exp": 1599098963,
+  "tenantId": "AzureFluidTenantId",
+  "ver": "1.0"
+}.[Signature]
+```
+
+Every request to FRS must be signed with a valid JWT. The FRS documentation contains additional details about [how to
+sign the token][1]. Fluid delegates the responsibility of creating and signing these tokens to a *token provider.*
+
+[1]: (https://github.com/MicrosoftDocs/azure-fluid-preview-pr/blob/main/azure-fluid-relay-preview-pr/articles/howtos/fluid-jwtoken.md#how-can-you-generate-an-azure-fluid-relay-token)
+
+{{% callout title="More information" %}}
+
+* [Introduction to JWTs](https://jwt.io/introduction)
+* [Payload claims in FRS](https://github.com/MicrosoftDocs/azure-fluid-preview-pr/blob/main/azure-fluid-relay-preview-pr/articles/howtos/fluid-jwtoken.md#payload-claims)
+* [Scopes in FRS](need a url)
+* [Signing requests](https://github.com/MicrosoftDocs/azure-fluid-preview-pr/blob/main/azure-fluid-relay-preview-pr/articles/howtos/fluid-jwtoken.md#how-can-you-generate-an-azure-fluid-relay-token)
+
+{{% /callout %}}
+
+## The token provider
+
+A token provider is responsible for creating and signing tokens that the Fluid runtime then uses to make requests to the
+service (FRS in this case). You are expected to provide your own token provider implementation for most scenarios.
+However, Fluid provides an `InsecureTokenProvider` that accepts your FRS tenant secret and returns signed tokens. This
+token provider is useful for testing, but in production scenarios you should use a more secure token provider.
+
+### A secure serverless token provider
+
+One option for building a secure token provider is to create a serverless Azure Function and expose it as a token
+provider. This enables you to store the FRS secret on a secure server. Your application calls the Function to generate
+tokens rather than signing them locally like the `InsecureTokenProvider` does.
+
+An example of such a function is available at <https://github.com/microsoft/FrsAzureFunctions>. There is a
+corresponding `FrsAzFunctionTokenProvider` for this Function in the `@fluid-experimental/frs-client` package.
```

Comment:
```suggestion
One option for building a secure token provider is to create a serverless Azure Function and expose it as a token
provider. This enables you to store the *tenant secret key* on a secure server. Your application calls the Function to generate tokens rather than signing them locally like the `InsecureTokenProvider` does.

An example of such a function is available at <https://github.com/microsoft/FrsAzureFunctions>. There is a
corresponding `FrsAzFunctionTokenProvider` for this Function in the `@fluid-experimental/frs-client` package.
```